### PR TITLE
Users can create Dev Env on a given branch

### PR DESCRIPTION
**Pull request by gtardif**
Signed-off-by: Guillaume Tardif <guillaume.tardif@gmail.com>

### Proposed changes

examples of how to specify a branch when creating a Dev Env. 
New, coming in next release (likely Desktop 3.5.2)

### CODE REVIEW

File: desktop/dev-environments.md [hunk 1]
```diff
@@ -56,6 +56,18 @@ You can launch the application by running the command `make run` in your VS Code
 
 ![Single container Dev environment](/images/dev-env-localhost.png){:width="700px"}
 
+### Create a Dev Environment from a specific branch
```
Comment (glours) [inline desktop/dev-environments.md:59]: We should at least mention `tag`

**Review by glours — CHANGES_REQUESTED** [review]

**Review by glours — APPROVED** [review]

**Review by usha-mandya — DISMISSED** [review]
LGTM

**Comment by gtardif** [comment]
@usha-mandya I added a note with a known bug we just saw with @rumpl (that will be fixed in the next release)

File: desktop/dev-environments.md [hunk 2]
```diff
@@ -56,6 +56,22 @@ You can launch the application by running the command `make run` in your VS Code
...
+
+ or
+
+ `git@github.com:dockersamples/single-dev-env.git@mybranch`
+
+Docker then clones the repository with your specified branch or tag.
+
+> **Note**
+>
+> Known bug: when using `https://` Git URLs, you need to remove the `.git` suffix at the end of the URL if you want to specify a branch with `@mybranch`.
```
Comment (usha-mandya) [inline desktop/dev-environments.md:73]:
```suggestion
> When cloning a Git branch using `https://`, you must remove the `.git` suffix at the end of the URL. For example, to clone `mybranch` on the `single-dev-env` Git repo, you must use `https://github.com/dockersamples/single-dev-env@mybranch`.
```

Comment (gtardif) [reply desktop/dev-environments.md:73]: OK. not calling this a known bug, it feels a bit strange that you need to remove the .git suffix for https, but not for ssh urls. Should we at least mention this is just "for now", and we’ll improve this later on ? 

Also, I didn’t mention an example, as this is the exact same one mentioned just above, not sure it’s worth repeating ?

Comment (gtardif) [reply desktop/dev-environments.md:73]: Could I rephrase: 
`Known bug: when cloning a Git branch using `https://`, you must remove the `.git` suffix at the end of the URL` ?

Comment (gtardif) [reply desktop/dev-environments.md:73]: (or "known limitation" ? )

Comment (usha-mandya) [reply desktop/dev-environments.md:73]: > "Could I rephrase:
Known bug: when cloning a Git branch using https://, you must remove the .git suffix at the end of the URL...

Sounds good. Can we change Known bug to "Known issue"?

Comment (gtardif) [reply desktop/dev-environments.md]: updated with "known issue" as suggested

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**